libs/testserver/server.go

```diff
@@ -39,22 +41,20 @@ type Request struct {
 }
 
 func New(t testutil.TestingT) *Server {
-	mux := http.NewServeMux()
-	server := httptest.NewServer(mux)
+	router := mux.NewRouter()
+	server := httptest.NewServer(router)
 	t.Cleanup(server.Close)
 
 	s := &Server{
 		Server:         server,
-		Mux:            mux,
+		Mux:            router,
```

Comment:
nit: rename `Mux` -> `Router` to reflect the object type.

Reply:
Makes sense, 016f5980ba97e06143039cf287b5f57bccf27dbe